__tests__/app.test.js

```diff
@@ -209,6 +208,160 @@ describe('/api/articles', () => {
+    return request(app)
+      .get('/api/articles')
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toBeSortedBy('created_at', { descending: true });
+      });
+  });
+  describe('QUERY: sort_by', () => {
+    test('GET: 200 - Sorts by sort_by query article_id', () => {
+      return request(app)
+        .get('/api/articles?sort_by=article_id')
+        .expect(200)
+        .then(({ body: { articles } }) => {
+          expect(articles).toBeSortedBy('article_id', { descending: true });
+        });
+    });
+    test('GET: 200 - Sorts by sort_by query author', () => {
+      return request(app)
+        .get('/api/articles?sort_by=author')
+        .expect(200)
+        .then(({ body: { articles } }) => {
+          expect(articles).toBeSortedBy('author', { descending: true });
+        });
+    });
+    test('GET: 200 - Sorts by sort_by query title', () => {
+      return request(app)
+        .get('/api/articles?sort_by=title')
+        .expect(200)
+        .then(({ body: { articles } }) => {
+          expect(articles).toBeSortedBy('title', { descending: true });
+        });
+    });
+    test('GET: 200 - Sorts by sort_by query topic', () => {
+      return request(app)
+        .get('/api/articles?sort_by=topic')
+        .expect(200)
+        .then(({ body: { articles } }) => {
+          expect(articles).toBeSortedBy('topic', { descending: true });
+        });
+    });
+    test('GET: 200 - Sorts by sort_by query created_at', () => {
+      return request(app)
+        .get('/api/articles?sort_by=created_at')
+        .expect(200)
+        .then(({ body: { articles } }) => {
+          expect(articles).toBeSortedBy('created_at', { descending: true });
+        });
+    });
+    test('GET: 200 - Sorts by sort_by query votes', () => {
+      return request(app)
+        .get('/api/articles?sort_by=votes')
+        .expect(200)
+        .then(({ body: { articles } }) => {
+          expect(articles).toBeSortedBy('votes', { descending: true });
+        });
+    });
+    test('GET: 200 - Sorts by sort_by query comment_count', () => {
+      return request(app)
+        .get('/api/articles?sort_by=comment_count')
+        .expect(200)
+        .then(({ body: { articles } }) => {
+          expect(articles).toBeSortedBy('comment_count', { descending: true });
+        });
+    });
+    test('GET: 400 - Responds with 400 if not a valid sort_by query', () => {
+      return request(app)
+        .get('/api/articles?sort_by=not_valid')
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe('Bad request');
+        });
+    });
+  });
+  describe('QUERY: order', () => {
+    test('GET: 200 - Sorts by ASC order when given and order query of asc', () => {
+      return request(app)
+        .get('/api/articles?order=asc')
+        .expect(200)
+        .then(({ body: { articles } }) => {
+          expect(articles).toBeSortedBy('created_at', { ascending: true });
```

Comment:
I don't think `{ ascending: true }` is specifically an option, I think this is just the default behaviour of `jest-sorted` as per the docs